paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -268,10 +270,19 @@ internal data class PaymentMethodMetadata(
                 sharedDataSpecs = sharedDataSpecs,
                 externalPaymentMethodSpecs = externalPaymentMethodSpecs,
                 paymentMethodSaveConsentBehavior = elementsSession.toPaymentSheetSaveConsentBehavior(),
+                linkConfiguration = null,
                 isGooglePayReady = isGooglePayReady,
             )
         }
 
+        internal fun PaymentMethodMetadata.withLinkConfiguration(
+            configuration: LinkConfiguration?
+        ): PaymentMethodMetadata {
+            return copy(
+                linkConfiguration = configuration,
+            )
+        }
+
```

Comment:
This is super hacky and I don't like it. We require the metadata in `PaymentSheetLoader` to create the `Link` configuration but we also require the `Link` configuration to properly create the metadata.